### ISSUE TITLE
entity.citation.present check

### ISSUE THREAD

**Issue by gothub** [NCEAS/metadig-checks#50]
## Description

Check if a data citation for attribution is present.

## Priority

- FAIR: Required

## Issues
- is the citation for the entire dataset or each entity?
- should a single citation field be searched for, or the fields that would be used to construct a citation. EDI uses creator/individualName, title, pubdate, publisher, alternateId[type="doi] to construct a citation.

## Procedure

- TBD

**Comment by cscully-allison** [comment]
Granular fields facilitates synthesis and interoperability and are better than a single citation.

**Comment by iamchrisser** [comment]
> Granular fields facilitates synthesis and interoperability and are better than a single citation.

Agreed, generally, but this gets tricky when different repos use different stylesheets to create citations from the granular fields. Maybe something like a 'preferred citation' field that is shown on the landing page, in addition to the individual fields (obvs).

**Comment by mbjones** [comment]
Good point @cscully-allison. I think that preferred citations are represented differently across metadata dialects.  Sometimes there is a specific field, and sometimes the specification expects a citation to be built from granular fields.  I think the check should honor both cases, and check to be sure the content needed for the preferred citation is present in the record, recognizing that different metadata dialects handle this in different ways.
